<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Raspodeli</class>
 <widget class="QDialog" name="Raspodeli">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Raspodela stavki</string>
  </property>
  <widget class="QTreeView" name="treeStavke">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>521</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>441</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Zaposleni:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboZaposleni"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Raspodeli</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="zapamti">
      <property name="text">
       <string>Izadji</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
